macros/src/types/tuple.rs

```diff
@@ -71,6 +71,15 @@ fn format_field(
     if skip {
         return Ok(());
     }
+    
+    let ty = if let Some(_type_as) = &type_as {
```

Comment:
Rename the variable to remove the `_` prefix, and if possible use `ref` instead of `&`